roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_rhel_kvm/assert-configuration.yml

```diff
@@ -4,11 +4,13 @@
 - name: Get kernel command line
   ansible.builtin.command: cat /proc/cmdline
   register: __sap_hypervisor_node_preconfigure_kernelcmdline_assert
+  changed_when: __sap_hypervisor_node_preconfigure_kernelcmdline_assert.rc != 0
 
 - name: "Assert - Kernel same page merging (KSM): Get status"
-  ansible.builtin.shell: systemctl status ksm
+  ansible.builtin.command: systemctl status ksm
```

Comment:
Use `ansible.builtin.systemd` instead?

Reply:
I would like to leave the rhel_kvm code as it is for now, but I'll keep that in my backlog.

Reply:
Then I'd drop this change for now.